handout/routing/redirects.md

```diff
@@ -0,0 +1,16 @@
+# Route Redirections #
```

Comment:
I think we should cover `pathMatch: 'full'` and `pathMatch: 'prefix'` - I can see this being a question that most people would ask after reading this section if we didn't touch on it.